oci/private/image.sh.tpl

```diff
@@ -151,7 +169,10 @@ if [ -n "${WORKDIR}" ]; then
 fi
 
 if [ -n "$OUTPUT" ]; then
+
     "${CRANE}" pull "${REF}" "./${OUTPUT}" --format=oci
```

Comment:
nit: maybe convince crane people to introduce a new flag called `--prune` to remove any unreferenced blobs.